### PR TITLE
New how to apply guidance

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/AWcZeVWr/3312-review-how-to-apply-for-teacher-training-page

### Context

As part of our in cycle conversion work, we want to provide better guidance for candidates around the application process. This involves updating our current how to apply page as well as creating new pages on the personal statement and references.

### CODE REVIEW

File: app/views/content/train-to-be-a-teacher/how-to-apply-for-teacher-training.md [hunk 1]
```diff
@@ -36,142 +38,106 @@ inset_text:
     title: Start your training this September
```
Comment (EllieNodder) [inline app/views/content/train-to-be-a-teacher/how-to-apply-for-teacher-training.md:38]: @plisovin this inset text isn't showing up in my preview app but I'm not sure why - do you mind taking a look when you can?

Comment (plisovin) [reply app/views/content/train-to-be-a-teacher/how-to-apply-for-teacher-training.md:38]: @EllieNodder are you talking about this text?

Screenshot 2023-07-04 at 12 19 20

Comment (EllieNodder) [reply app/views/content/train-to-be-a-teacher/how-to-apply-for-teacher-training.md:38]: no, this text: 
image

**Review by philspray — APPROVED** [review]
Looks great to me!

**Pull request merged**